contracts/Account.cairo

```diff
@@ -27,6 +31,20 @@ struct Message:
     member nonce: felt
 end
 
+struct MCall:
```

Comment:
The way I see it is that a `MCall` is a "modified" `Call` that we introduce until array of `Call`s can be passed to the `__execute__` method. So I would prefer a name that both highlight that it is a temporary solution and at the same time be close enough to the target naming (i.e. `Call`). I do feel that `MCall` does that job.

If we were to call it `MultiCall` I fail to see what would be the name of the target `Call`. Also (and again, this is only how I see it) to me the multicall is the array of `Call`, not the individual calls.

All this to say that I would prefer to call is `MCall` over `MultiCall` :-)

Selectors in solidity are computed based on the method name and the argument types so it should not change the EIP165 interface ID.

Reply:
The only problem I see with `MCall` is that it's not clear what the _M_ stands for, which can be confusing. `MultiCall` is more clear than `ModifiedCall` to me. What about `CallList` or `CallArray`? That would be both explicit and indicative of the temporary nature of the struct (since it's directly linked to the lack of struct arrays).

Reply:
I've renamed `MCall` to `CallArray`. Let me know what you think.